src/app/components/surveillance/complaint/complaint.html

```diff
@@ -1,3 +1,141 @@
 <div id="chpl-complaint" ng-form="$ctrl.form">
-  Complaint "dumb component"
+  <div ng-if="$ctrl.mode === $ctrl.modes.SELECT">
+    <table class="table table-condensed">
+      <thead>
+        <tr>
+          <th scope="col">ONC-ACB</th>
+          <th scope="col">Type</th>
+          <th scope="col">Status</th>
+          <th scope="col">ONC Complaint Id</th>
```

Comment:
```suggestion
          <th scope="col">ONC Complaint ID</th>
```

There are about seven "Id"s in our codebase that I can find, and dozens of "ID"s. Would like to try to keep consistent